ringvax/app.py

```diff
@@ -180,60 +182,102 @@ def app():
         f"Ran {nsim} simulations in {format_duration(toc - tic)} with an $R_0$ of {infectious_duration * infection_rate:.2f} (the product of the average duration of infection and the infectious rate)."
     )
 
-    tab1, tab2 = st.tabs(["Simulation summary", "Per-simulation results"])
-    with tab1:
-        sim_df = get_all_person_properties(sims)
+    n_at_max = sum(
+        1 for sim in sims if sim.termination["criterion"] == "max_infections"
+    )
 
-        pr_control = prob_control_by_gen(sim_df, control_generations)
-        st.header(
-            f"Probability of control: {pr_control:.0%}",
-            help=f"The probability that there are no infections in the {format_control_gens(control_generations)}, or equivalently that the {format_control_gens(control_generations - 1)} do not produce any further infections.",
+    show = True if n_at_max == 0 else False
```

Comment:
I think there's some way to refactor all this for mild cleanliness; I'll open a PR if I have something useful to say

Reply:
Yeah, that logic is definitely not the prettiest, happy if there's a cleaner way to express "plot the results unless the user pushes the big red override button"